app/facades/quote_facade.rb

```diff
@@ -0,0 +1,9 @@
+class QuoteFacade
+    def self.get_quotes(keyword)
+      quotes_data = QuoteService.quote_search(keyword)
+      
+      quotes_data[:results][0..9].map do |quote_data|
+        Quote.new(quote_data)
+      end
+    end
```

Comment:
In order to solve the issue of duplicate API calls in the controller, I would return either an array or a hash that includes all the data you want in the controller that comes from this one API call. That could look something like this:
```ruby
    def self.get_quotes(keyword)
      quotes_data = QuoteService.quote_search(keyword)

      quotes = quotes_data[:results][0..9].map do |quote_data|
        Quote.new(quote_data)
      end
      [quotes_data, quotes]
    end
```
and then in the controller, you would just update to:
```ruby
@data, @quotes = QuoteFacade.get_quotes(params[:query])
```